src/partials/tablecontent.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react'
-function tablecontent (props) {
+import {Link} from 'react-router-dom'
+import loadingicon from '../loadingicon.gif'
+function Tablecontent (props) {
```

Comment:
@phuongdk Use explicit variable name instead of `props`. Or just use:
```
function Tablecontent (props) {}
```
=>
```
function Tablecontent ({ loading, movies, ... }) {}
```